spec/features/reverse_peeps_spec.rb

```diff
@@ -0,0 +1,7 @@
+feature 'Reversing chronological order of peeps' do
+  scenario 'visiting the index page and reversing order of peeps' do
+    visit('/')
+    click_button("Reverse Chronological Order")
+    # TODO: how to assert on order reversing?
+  end
+end
```

Comment:
Great to see you considering how a test can be executed, shows a great use of TDD. You could maybe do this test by setting a default order of peeps initially, checking they are in the correct order by indexing, click your reverse button and checking their index has changed perhaps? Could look at this when we have a catch up if you like?